pages/linux/ark.md

```diff
@@ -0,0 +1,16 @@
+# ark
+
+> KDE archiving tool.
+> More information: <https://docs.kde.org/stable5/en/kdeutils/ark/index.html>.
+
+- Extract an archive into the current directory:
+
+`ark --batch {{archive}}`
+
+- Change extraction directory:
+
+`ark --batch --destination {{path/to/directory}} {{archive}}`
+
+- Create archive if it does not exist and add files to it:
```

Comment:
The same as sbrl suggested. Perhaps this will read better? 
```suggestion
- Create an archive if it does not exist and add files to it:
```

Reply:
Right, I'll update that.